### ISSUE TITLE
`PrimeSet::prime_factors()` prints factors to `stdout`

### ISSUE THREAD

**Issue by Ontonator** [wackywendell/primes#3]
Code:
```rust
PrimeSet::new().prime_factors(999990).for_each(drop);
```
`stdout`:
```
Pushing 2 (999990 / 999990)
Pushing 3 (499995 / 999990)
Pushing 3 (166665 / 999990)
Pushing 5 (55555 / 999990)
Pushing 41 (11111 / 999990)
Final push 41 (271 / 999990)
```

These seem like debug messages that have just been left in, but if that is not the case, it would be better to have a parameter or feature to enable/disable the printing.

**Comment by wackywendell** [comment]
Oops!

**Comment by wackywendell** [comment]
Fixed in 26e0a5cfcd6698c67b4c7bd773ea2eb068b930cb / version 0.2.4 on crates.io.
